native/src/components/TtsContainer.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { createContext, ReactElement, useCallback, useEffect, useMemo, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { AppState } from 'react-native'
+import Tts from 'react-native-tts'
+
+import { truncate } from 'shared/utils/getExcerpt'
+
+import { AppContext } from '../contexts/AppContextProvider'
+import { reportError } from '../utils/sentry'
+import TtsPlayer from './TtsPlayer'
+
+const MAX_TITLE_DISPLAY_CHARS = 20
+
+export type TtsContextType = {
+  enabled: boolean
+  setEnabled: (enable: boolean) => void
+  visible: boolean
+  setVisible: (visible: boolean) => void
+  sentences: string[] | null
+  setSentences: (sentences: string[]) => void
+}
+
+export const ttsContext = createContext<TtsContextType>({
```

Comment:
:wrench: So far we are using upper case names for contexts, please change accoringly
```suggestion
export const TtsContext = createContext<TtsContextType>({
```